### PR TITLE
Adding config for development

### LINKED ISSUE #155: Provide a temporal config for development
Because now config is stored globally, when trying things in dev, you own gaucho config may be affected.

Maybe with a DEV flag or something it would be possible to connect to a different storage than the plain gaucho storage

### PR DESCRIPTION
**Pull request by tdeschryver**
## Description of the PR
Using a dev config when running in dev mode.

### Issues Related
Closes #155.

**Checklist**
- [x] Your PR is done against the `dev` branch
- [x] Your origin branch is updated with the latest changes from `dev`
- [ ] All the tests and jshint are passing (`npm run test`) - this is giving errors for me, tests are passing when running `mocha` 
- [x] Changelog is updated with a brief description of your changes (if required)
- [ ] Your changes have tests - don't know how to set up electron tests to be honest. Once #94 is resolved, I can add tests.

### CODE REVIEW

File: app/common/app_config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 const Store = require('electron-store');
-
+const { isDevEnv } = require('./utils');
```
Comment (angrykoala) [inline app/common/app_config.js]: Style detail, while this way is generally accepted as a good syntax for requires, I'm using a more basic syntax in the rest of the project (partially for personal preference to be honest), just for consistency. Could you change this for a syntax according to the rest of the code?
Something like this
```js
const utils=require('./utils');
// Rest of the code
utils.isDevEnv()
```

Thanks

File: main.js [hunk 2]
```diff
@@ -4,11 +4,7 @@ const path = require('path');
 
 const MainWindow = require('./app/main/main_window');
 const AppEvents = require('./app/main/app_events');
-
-
-function isDevEnv() {
-    return process.env.NODE_ENV === "dev";
-}
+const { isDevEnv } = require('./app/common/utils');
```
Comment (angrykoala) [inline main.js]: ditto to the first comment

File: package.json [hunk 3]
```diff
@@ -13,21 +13,22 @@
   },
   "devDependencies": {
     "chai": "^4.0.2",
+    "cross-env": "^5.0.5",
```
Comment (angrykoala) [inline package.json]: This is just a question, I understand this is to provide a cross-compatible env, but afaik, envs also work in windows. What is the case we need this?

Comment (tdeschryver) [reply package.json]: When I run `npm run start-dev` without `cross-env` I get the following error:
`'NODE_ENV' is not recognized as an internal or external command,`

File: app/common/app_config.js (hunk 1 above)
Comment (tdeschryver) [reply app/common/app_config.js]: Yup, no problemo!

File: package.json (hunk 3 above)
Comment (angrykoala) [reply package.json]: out of curiosity, What OS are you using?

I agree is better to use cross-env to avoid this kind of problems

Comment (tdeschryver) [reply package.json]: Windows 7

**Pull request merged**